sweep/tx_input_set.go

```diff
@@ -179,11 +188,28 @@ func (t *txInputSet) addToState(inp input.Input,
 
 	// If the input comes with a required tx out that is below dust, we
 	// won't add it.
+	//
+	// NOTE: only HtlcSecondLevelAnchorInput returns non-nil RequiredTxOut.
 	reqOut := inp.RequiredTxOut()
 	if reqOut != nil {
 		// Fetch the dust limit for this output.
+		//
+		// TODO(yy): replace this check with `mempool.IsDust` to take
+		// the minRelayTxFee into account in case it's not using the
```

Comment:
min relay fee and dust are unrelated

Reply:
updated the docs to make it more clear - think we should also update it in `btcd.mempool`.